science/dftd3/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                dftd3
+version             3.2.0
+revision            0
+categories          science physics chemistry
+license             public-domain
+maintainers         {@kamischi web.de:karl-michael.schindler} openmaintainer
+description         Dispersion Correction For Quantum Chemical Methods
+long_description    Dispersion correction for density functionals, Hartree-Fock \
+                    and semi-empirical quantum chemical methods
+homepage            https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3
+master_sites        ${homepage}
+distfiles           dftd3.tgz
+checksums           md5     a76c752e587422c239c99109547516d2 \
```

Comment:
`md5` is no longer used as a checksum type. This `md5` checksum should be removed.

Reply:
Done as suggested.